lib/pal-platform/Middleware/curl/curl.cmake

```diff
@@ -15,18 +15,18 @@
 #################################################################################
 
 if (NOT MBED_CLOUD_CLIENT_CURL_DYNAMIC_LINK)
-    set (CMAKE_USE_OPENSSL OFF CACHE BOOL "disable openssl" FORCE)
+    set (CURL_USE_OPENSSL OFF CACHE BOOL "disable openssl" FORCE)
```

Comment:
Are those CMake flags used somewhere?
I would expect to see another rename where this flag is used

Reply:
I'm not sure I understand @jenia81  - this is a flag used for controlling the build parameters for CMAKE. They have renamed that parameters in 7.81 release (commit had link to PR), so that what was (misleadingly to be honest) previously called CMAKE_USE_OPENSSL is now called CURL_USE_OPENSSL, sort of a namespace issue.
- So the curl library building (if building it from source - would use that flag).

Reply:
Copy pasta from that commit; curl.cmake - CURL_USE_LIBSSH2 (not CMAKE_USE)
Same as with the CURL_USE_OPENSSL, too. This define has been changed
in curl via pull https://github.com/curl/curl/pull/8044/files
so that CMAKE_USE_LIBSSH2 -> CURL_USE_LIBSSH2

Reply:
So, these should get used with a statically linked `libcurl`.

Reply:
@jenia81 - all OK with this?

Reply:
yep